src/tags/include_tag.rs

```diff
@@ -0,0 +1,76 @@
+
+use Renderable;
+use context::Context;
+use token::Token;
+use LiquidOptions;
+use template::Template;
+use parser;
+use lexer;
+use error::{Result, Error};
+
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+
+struct Include {
+    partial: Template,
+}
+
+impl Renderable for Include {
+    fn render(&self, mut context: &mut Context) -> Result<Option<String>> {
+        self.partial.render(&mut context)
+    }
+}
+
+fn parse_partial(path: &str, options: &LiquidOptions) -> Result<Template> {
+    let path = Path::new(&path);
+
+    // check if file exists
+    if !path.exists() {
+        return Error::parser_msg(&format!("{:?} does not exist", path));
+    }
```

Comment:
you can leave this out I think, if the path doesn't exist `open` will throw a similar error